apps/web/src/components/balances/AssetsTable/index.tsx

```diff
@@ -157,6 +172,11 @@ const AssetsTable = ({
               collapsed: item.tokenInfo.address === hidingAsset,
               content: <FiatBalance balanceItem={item} />,
             },
+            change: {
+              rawValue: Number(item.fiatBalance24hChange ?? -9999999999999999),
```

Comment:
I think sorting will work as expected if we use `null`. If not, we should add a comment re. the arbitrary number. We could also use `-1 * Number.MAX_SAFE_INTEGER`. What do you think?

Reply:
The current comparator handled `null` the same as `0` so they would always get sorted between positive and negative relative changes.

I adjusted the comparator to always sort null / undefined last.